qafoo-profiler package 403  Forbidden

Hi,

There is an issue with tideways when build the docker, it return an error 

E: Failed to fetch http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages/dists/debian/main/binary-amd64/Packages  403  Forbidden

You can find the full log in following

```
2020-11-05T09:55:16.4574573Z Sending build context to Docker daemon  113.2kB
2020-11-05T09:55:16.4575332Z 
2020-11-05T09:55:16.5067834Z [91m[WARNING]: Empty continuation line found in:
2020-11-05T09:55:16.5093334Z     RUN curl -sL https://deb.nodesource.com/setup_11.x > /tmp/install-node.sh  && bash /tmp/install-node.sh  && apt-get update -qq -y  && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install     nodejs     rsyslog     sudo     php-sqlite3   && npm config set --global loglevel warn  && npm install --global marked  && npm install --global node-gyp  && npm install --global yarn   && npm rebuild node-sass   && apt-get install nano  -y  && apt-get install php7.2-imap -y && sudo apt-get update && apt-get install gcc -y && apt-get install build-essential -y && apt-get install php7.2-dev -y && apt-get install ghostscript -y && wget https://github.com/maxmind/libmaxminddb/releases/download/1.4.2/libmaxminddb-1.4.2.tar.gz && tar -zxvf libmaxminddb-1.4.2.tar.gz && cd libmaxminddb-1.4.2 && ./configure && make && make check && make install && sudo ldconfig && cd .. && git clone https://github.com/maxmind/MaxMind-DB-Reader-php.git && cd MaxMind-DB-Reader-php/ext && /usr/bin/phpize && ./configure && make && make install && echo "extension=maxminddb.so" >> /etc/php/7.2/cli/php.ini && echo "extension=maxminddb.so" >> /etc/php/7.2/fpm/php.ini && sudo apt install openssl && curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && sudo apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget curl php7.2-curl && sudo npm install --global --unsafe-perm puppeteer && sudo chmod -R o+rx /usr/lib/node_modules/puppeteer/.local-chromium && sed 's|mozilla\/AddTrust_External_Root.crt|#mozilla\/AddTrust_External_Root.crt|g' -i /etc/ca-certificates.conf && sudo update-ca-certificates -f -v && wget -q -O - https://packages.blackfire.io/gpg.key | apt-key add - && echo "deb http://packages.blackfire.io/debian any main" | sudo tee /etc/apt/sources.list.d/blackfire.list && apt update -y && apt install blackfire-agent -y && version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$version && mkdir -p /tmp/blackfire && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire && mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so && printf "extension=blackfire.so\n" > /etc/php/7.2/cli/conf.d/blackfire.ini && printf "extension=blackfire.so\n" > /etc/php/7.2/fpm/conf.d/blackfire.ini && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz && curl -sO https://gordalina.github.io/cachetool/downloads/cachetool.phar && chmod +x cachetool.phar && mv cachetool.phar /usr/local/bin/cachetool && echo "nameserver 8.8.8.8" >> /etc/resolv.conf && echo "nameserver 8.8.4.4" >> /etc/resolv.conf && apt-get install dnsmasq -y && groupadd -r dnsmasq && composer self-update && composer self-update --2 && apt-get install jpegoptim -y && apt-get install optipng -y && apt-get install pngquant -y && npm install -g svgo && apt-get install gifsicle -y && apt-get install webp -y && apt-get remove -qq -y php7.2-dev pkg-config gcc libmagickwand-dev build-essential && apt-get auto-remove -qq -y && apt-get clean && rm -rf /etc/ImageMagick-6/policy.xml && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
2020-11-05T09:55:16.5111491Z [WARNING]: Empty continuation lines will become errors in a future release.
2020-11-05T09:55:16.5113203Z [0mStep 1/8 : FROM quay.io/continuouspipe/php7.2-nginx:latest
2020-11-05T09:55:16.6977528Z latest: Pulling from continuouspipe/php7.2-nginx
2020-11-05T09:55:16.6987296Z 4f53fa4d2cf0: Pulling fs layer
2020-11-05T09:55:16.6990593Z 6af7c939e38e: Pulling fs layer
2020-11-05T09:55:16.6993707Z 903d0ffd64f6: Pulling fs layer
2020-11-05T09:55:16.6996331Z 04feeed388b7: Pulling fs layer
2020-11-05T09:55:16.7025129Z f005f75618ea: Pulling fs layer
2020-11-05T09:55:16.7025654Z 04feeed388b7: Waiting
2020-11-05T09:55:16.7026154Z d642c4b77f87: Pulling fs layer
2020-11-05T09:55:16.7026609Z 1aa4d243441a: Pulling fs layer
2020-11-05T09:55:16.7027141Z 3dca63a2f574: Pulling fs layer
2020-11-05T09:55:16.7029814Z d5f45f600b62: Pulling fs layer
2020-11-05T09:55:16.7030571Z 8e06cbc8db0d: Pulling fs layer
2020-11-05T09:55:16.7031039Z f166d95ae08c: Pulling fs layer
2020-11-05T09:55:16.7031449Z b3f416117534: Pulling fs layer
2020-11-05T09:55:16.7031834Z f005f75618ea: Waiting
2020-11-05T09:55:16.7032208Z d642c4b77f87: Waiting
2020-11-05T09:55:16.7032639Z 1aa4d243441a: Waiting
2020-11-05T09:55:16.7033080Z 3dca63a2f574: Waiting
2020-11-05T09:55:16.7033444Z d5f45f600b62: Waiting
2020-11-05T09:55:16.7033895Z 8e06cbc8db0d: Waiting
2020-11-05T09:55:16.7034229Z f166d95ae08c: Waiting
2020-11-05T09:55:16.7034577Z 70a35acfc48c: Pulling fs layer
2020-11-05T09:55:16.7034941Z 5e226504f1b2: Pulling fs layer
2020-11-05T09:55:16.7035281Z 977a2739955b: Pulling fs layer
2020-11-05T09:55:16.7035597Z b3f416117534: Waiting
2020-11-05T09:55:16.7035911Z 70a35acfc48c: Waiting
2020-11-05T09:55:16.7036225Z 5e226504f1b2: Waiting
2020-11-05T09:55:16.7036576Z e3fdf882a11d: Pulling fs layer
2020-11-05T09:55:16.7037018Z e3fdf882a11d: Waiting
2020-11-05T09:55:16.7037332Z 977a2739955b: Waiting
2020-11-05T09:55:16.8054369Z 903d0ffd64f6: Verifying Checksum
2020-11-05T09:55:16.8058448Z 903d0ffd64f6: Download complete
2020-11-05T09:55:16.8061508Z 6af7c939e38e: Verifying Checksum
2020-11-05T09:55:16.8062262Z 6af7c939e38e: Download complete
2020-11-05T09:55:16.8812981Z 04feeed388b7: Verifying Checksum
2020-11-05T09:55:16.8814010Z 04feeed388b7: Download complete
2020-11-05T09:55:17.1108823Z d642c4b77f87: Verifying Checksum
2020-11-05T09:55:17.1109350Z d642c4b77f87: Download complete
2020-11-05T09:55:17.2892603Z 1aa4d243441a: Verifying Checksum
2020-11-05T09:55:17.2893179Z 1aa4d243441a: Download complete
2020-11-05T09:55:17.8683893Z 4f53fa4d2cf0: Verifying Checksum
2020-11-05T09:55:17.8685791Z 4f53fa4d2cf0: Download complete
2020-11-05T09:55:17.8686377Z f005f75618ea: Verifying Checksum
2020-11-05T09:55:17.8686917Z f005f75618ea: Download complete
2020-11-05T09:55:17.8687458Z d5f45f600b62: Verifying Checksum
2020-11-05T09:55:17.8688276Z d5f45f600b62: Download complete
2020-11-05T09:55:17.8689357Z 8e06cbc8db0d: Verifying Checksum
2020-11-05T09:55:17.8690159Z 8e06cbc8db0d: Download complete
2020-11-05T09:55:17.8690871Z f166d95ae08c: Verifying Checksum
2020-11-05T09:55:17.8692001Z f166d95ae08c: Download complete
2020-11-05T09:55:17.8692433Z b3f416117534: Verifying Checksum
2020-11-05T09:55:17.8692850Z b3f416117534: Download complete
2020-11-05T09:55:17.8693265Z 5e226504f1b2: Verifying Checksum
2020-11-05T09:55:17.8694484Z 5e226504f1b2: Download complete
2020-11-05T09:55:17.8910149Z 977a2739955b: Verifying Checksum
2020-11-05T09:55:17.8911894Z 977a2739955b: Download complete
2020-11-05T09:55:17.9488686Z 70a35acfc48c: Verifying Checksum
2020-11-05T09:55:17.9489462Z 70a35acfc48c: Download complete
2020-11-05T09:55:18.0540325Z e3fdf882a11d: Verifying Checksum
2020-11-05T09:55:18.0540840Z e3fdf882a11d: Download complete
2020-11-05T09:55:19.0350348Z 3dca63a2f574: Verifying Checksum
2020-11-05T09:55:19.0351179Z 3dca63a2f574: Download complete
2020-11-05T09:55:19.5920017Z 4f53fa4d2cf0: Pull complete
2020-11-05T09:55:19.6687464Z 6af7c939e38e: Pull complete
2020-11-05T09:55:19.7287586Z 903d0ffd64f6: Pull complete
2020-11-05T09:55:19.7817664Z 04feeed388b7: Pull complete
2020-11-05T09:55:21.5280880Z f005f75618ea: Pull complete
2020-11-05T09:55:21.5856136Z d642c4b77f87: Pull complete
2020-11-05T09:55:21.6498700Z 1aa4d243441a: Pull complete
2020-11-05T09:55:27.3827138Z 3dca63a2f574: Pull complete
2020-11-05T09:55:27.9287709Z d5f45f600b62: Pull complete
2020-11-05T09:55:29.3357355Z 8e06cbc8db0d: Pull complete
2020-11-05T09:55:30.1821219Z f166d95ae08c: Pull complete
2020-11-05T09:55:30.6347295Z b3f416117534: Pull complete
2020-11-05T09:55:30.8204132Z 70a35acfc48c: Pull complete
2020-11-05T09:55:30.8846645Z 5e226504f1b2: Pull complete
2020-11-05T09:55:30.9453028Z 977a2739955b: Pull complete
2020-11-05T09:55:31.0025719Z e3fdf882a11d: Pull complete
2020-11-05T09:55:31.0057316Z Digest: sha256:b5cb3d611aeaa9d0822116d63aa53474b2e15cd79d766f6cdbbb339e5e1b6f62
2020-11-05T09:55:31.0073279Z Status: Downloaded newer image for quay.io/continuouspipe/php7.2-nginx:latest
2020-11-05T09:55:31.0081965Z  ---> 66c86858a7d9
2020-11-05T09:55:31.0082298Z Step 2/8 : RUN mkdir /salla
2020-11-05T09:55:34.8214390Z  ---> Running in d7d7fe5d92a2
2020-11-05T09:55:36.2821396Z Removing intermediate container d7d7fe5d92a2
2020-11-05T09:55:36.2822925Z  ---> ea82729ea09d
2020-11-05T09:55:36.2823485Z Step 3/8 : COPY ./build/ /salla/
2020-11-05T09:55:37.2882125Z  ---> 6c1a50ded277
2020-11-05T09:55:37.2882862Z Step 4/8 : RUN chown -R build:build /salla
2020-11-05T09:55:37.3164516Z  ---> Running in 0faccb498ba7
2020-11-05T09:55:38.6776522Z Removing intermediate container 0faccb498ba7
2020-11-05T09:55:38.6777767Z  ---> 5c6babd095d3
2020-11-05T09:55:38.6799072Z Step 5/8 : RUN curl -sL https://deb.nodesource.com/setup_11.x > /tmp/install-node.sh  && bash /tmp/install-node.sh  && apt-get update -qq -y  && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install     nodejs     rsyslog     sudo     php-sqlite3   && npm config set --global loglevel warn  && npm install --global marked  && npm install --global node-gyp  && npm install --global yarn   && npm rebuild node-sass   && apt-get install nano  -y  && apt-get install php7.2-imap -y && sudo apt-get update && apt-get install gcc -y && apt-get install build-essential -y && apt-get install php7.2-dev -y && apt-get install ghostscript -y && wget https://github.com/maxmind/libmaxminddb/releases/download/1.4.2/libmaxminddb-1.4.2.tar.gz && tar -zxvf libmaxminddb-1.4.2.tar.gz && cd libmaxminddb-1.4.2 && ./configure && make && make check && make install && sudo ldconfig && cd .. && git clone https://github.com/maxmind/MaxMind-DB-Reader-php.git && cd MaxMind-DB-Reader-php/ext && /usr/bin/phpize && ./configure && make && make install && echo "extension=maxminddb.so" >> /etc/php/7.2/cli/php.ini && echo "extension=maxminddb.so" >> /etc/php/7.2/fpm/php.ini && sudo apt install openssl && curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && sudo apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget curl php7.2-curl && sudo npm install --global --unsafe-perm puppeteer && sudo chmod -R o+rx /usr/lib/node_modules/puppeteer/.local-chromium && sed 's|mozilla\/AddTrust_External_Root.crt|#mozilla\/AddTrust_External_Root.crt|g' -i /etc/ca-certificates.conf && sudo update-ca-certificates -f -v && wget -q -O - https://packages.blackfire.io/gpg.key | apt-key add - && echo "deb http://packages.blackfire.io/debian any main" | sudo tee /etc/apt/sources.list.d/blackfire.list && apt update -y && apt install blackfire-agent -y && version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$version && mkdir -p /tmp/blackfire && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire && mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so && printf "extension=blackfire.so\n" > /etc/php/7.2/cli/conf.d/blackfire.ini && printf "extension=blackfire.so\n" > /etc/php/7.2/fpm/conf.d/blackfire.ini && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz && curl -sO https://gordalina.github.io/cachetool/downloads/cachetool.phar && chmod +x cachetool.phar && mv cachetool.phar /usr/local/bin/cachetool && echo "nameserver 8.8.8.8" >> /etc/resolv.conf && echo "nameserver 8.8.4.4" >> /etc/resolv.conf && apt-get install dnsmasq -y && groupadd -r dnsmasq && composer self-update && composer self-update --2 && apt-get install jpegoptim -y && apt-get install optipng -y && apt-get install pngquant -y && npm install -g svgo && apt-get install gifsicle -y && apt-get install webp -y && apt-get remove -qq -y php7.2-dev pkg-config gcc libmagickwand-dev build-essential && apt-get auto-remove -qq -y && apt-get clean && rm -rf /etc/ImageMagick-6/policy.xml && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
2020-11-05T09:55:38.7043967Z  ---> Running in 5b75a441be42
2020-11-05T09:55:39.3559142Z 
2020-11-05T09:55:39.3560585Z ================================================================================
2020-11-05T09:55:39.3560935Z ================================================================================
2020-11-05T09:55:39.3561141Z 
2020-11-05T09:55:39.3561499Z                               DEPRECATION WARNING                            
2020-11-05T09:55:39.3561800Z 
2020-11-05T09:55:39.3562186Z   Node.js 11.x is no longer actively supported!
2020-11-05T09:55:39.3562501Z 
2020-11-05T09:55:39.3563002Z   You will not receive security or critical stability updates for this version.
2020-11-05T09:55:39.3563452Z 
2020-11-05T09:55:39.3563924Z   You should migrate to a supported version of Node.js as soon as possible.
2020-11-05T09:55:39.3564675Z   Use the installation script that corresponds to the version of Node.js you
2020-11-05T09:55:39.3565350Z   wish to install. e.g.
2020-11-05T09:55:39.3565561Z 
2020-11-05T09:55:39.3566742Z    * https://deb.nodesource.com/setup_10.x — Node.js 10 LTS "Dubnium"
2020-11-05T09:55:39.3567802Z    * https://deb.nodesource.com/setup_12.x — Node.js 12 LTS "Erbium" (recommended)
2020-11-05T09:55:39.3568871Z    * https://deb.nodesource.com/setup_14.x — Node.js 14 LTS "Fermium"
2020-11-05T09:55:39.3570088Z    * https://deb.nodesource.com/setup_15.x — Node.js 15 "Fifteen"
2020-11-05T09:55:39.3570577Z 
2020-11-05T09:55:39.3571099Z   Please see https://github.com/nodejs/Release for details about which
2020-11-05T09:55:39.3571737Z   version may be appropriate for you.
2020-11-05T09:55:39.3572039Z 
2020-11-05T09:55:39.3572525Z   The NodeSource Node.js distributions repository contains
2020-11-05T09:55:39.3573268Z   information both about supported versions of Node.js and supported Linux
2020-11-05T09:55:39.3574025Z   distributions. To learn more about usage, see the repository:
2020-11-05T09:55:39.3574732Z     https://github.com/nodesource/distributions
2020-11-05T09:55:39.3575135Z 
2020-11-05T09:55:39.3575401Z ================================================================================
2020-11-05T09:55:39.3575806Z ================================================================================
2020-11-05T09:55:39.3576020Z 
2020-11-05T09:55:39.3576597Z Continuing in 20 seconds ...
2020-11-05T09:55:39.3576858Z 
2020-11-05T09:55:59.3578723Z 
2020-11-05T09:55:59.3579791Z ## Installing the NodeSource Node.js 11.x repo...
2020-11-05T09:55:59.3580412Z 
2020-11-05T09:55:59.3616800Z 
2020-11-05T09:55:59.3618467Z ## Populating apt-get cache...
2020-11-05T09:55:59.3618962Z 
2020-11-05T09:55:59.3625742Z + apt-get update
2020-11-05T09:55:59.4264128Z Get:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [109 kB]
2020-11-05T09:55:59.5507647Z Get:2 http://archive.ubuntu.com/ubuntu xenial InRelease [247 kB]
2020-11-05T09:55:59.5513387Z Get:3 http://ppa.launchpad.net/ondrej/php/ubuntu xenial InRelease [23.9 kB]
2020-11-05T09:55:59.5833145Z Get:4 http://security.ubuntu.com/ubuntu xenial-security/main amd64 Packages [1841 kB]
2020-11-05T09:55:59.6273200Z Ign:5 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian InRelease
2020-11-05T09:55:59.6801990Z Get:6 http://security.ubuntu.com/ubuntu xenial-security/restricted amd64 Packages [15.9 kB]
2020-11-05T09:55:59.6820583Z Get:7 http://security.ubuntu.com/ubuntu xenial-security/universe amd64 Packages [968 kB]
2020-11-05T09:55:59.6957223Z Get:8 http://security.ubuntu.com/ubuntu xenial-security/multiverse amd64 Packages [9249 B]
2020-11-05T09:55:59.6992743Z Ign:9 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian Release
2020-11-05T09:55:59.7711481Z Ign:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:55:59.8476319Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:55:59.8906817Z Get:12 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 Packages [137 kB]
2020-11-05T09:55:59.9198792Z Ign:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:55:59.9576464Z Get:13 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [109 kB]
2020-11-05T09:55:59.9920956Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:56:00.0619791Z Get:14 http://archive.ubuntu.com/ubuntu xenial-backports InRelease [107 kB]
2020-11-05T09:56:00.0639753Z Ign:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:56:00.1354794Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:56:00.1410798Z Get:15 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages [1558 kB]
2020-11-05T09:56:00.2069376Z Ign:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:56:00.2783114Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:56:00.3497162Z Ign:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:56:00.3595169Z Get:16 http://archive.ubuntu.com/ubuntu xenial/restricted amd64 Packages [14.1 kB]
2020-11-05T09:56:00.3627790Z Get:17 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages [9827 kB]
2020-11-05T09:56:00.4211127Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:56:00.4931759Z Err:10 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main amd64 Packages
2020-11-05T09:56:00.4937574Z   403  Forbidden
2020-11-05T09:56:00.5661538Z Ign:11 http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian/main all Packages
2020-11-05T09:56:00.6585893Z Get:18 http://archive.ubuntu.com/ubuntu xenial/multiverse amd64 Packages [176 kB]
2020-11-05T09:56:00.6655954Z Get:19 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages [2353 kB]
2020-11-05T09:56:00.7235510Z Get:20 http://archive.ubuntu.com/ubuntu xenial-updates/restricted amd64 Packages [16.4 kB]
2020-11-05T09:56:00.7257676Z Get:21 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages [1512 kB]
2020-11-05T09:56:00.7614435Z Get:22 http://archive.ubuntu.com/ubuntu xenial-updates/multiverse amd64 Packages [26.7 kB]
2020-11-05T09:56:00.7627106Z Get:23 http://archive.ubuntu.com/ubuntu xenial-backports/main amd64 Packages [10.9 kB]
2020-11-05T09:56:00.7636815Z Get:24 http://archive.ubuntu.com/ubuntu xenial-backports/universe amd64 Packages [12.6 kB]
2020-11-05T09:56:01.7928008Z Fetched 19.1 MB in 2s (7962 kB/s)
2020-11-05T09:56:02.7377054Z Reading package lists...
2020-11-05T09:56:02.7641299Z [91mW: The repository 'http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian Release' does not have a Release file.
2020-11-05T09:56:02.7643285Z E: Failed to fetch http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages/dists/debian/main/binary-amd64/Packages  403  Forbidden
2020-11-05T09:56:02.7644381Z E: Some index files failed to download. They have been ignored, or old ones used instead.
2020-11-05T09:56:02.7645182Z [0mError executing command, exiting
2020-11-05T09:56:03.2964356Z The command '/bin/sh -c curl -sL https://deb.nodesource.com/setup_11.x > /tmp/install-node.sh  && bash /tmp/install-node.sh  && apt-get update -qq -y  && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install     nodejs     rsyslog     sudo     php-sqlite3   && npm config set --global loglevel warn  && npm install --global marked  && npm install --global node-gyp  && npm install --global yarn   && npm rebuild node-sass   && apt-get install nano  -y  && apt-get install php7.2-imap -y && sudo apt-get update && apt-get install gcc -y && apt-get install build-essential -y && apt-get install php7.2-dev -y && apt-get install ghostscript -y && wget https://github.com/maxmind/libmaxminddb/releases/download/1.4.2/libmaxminddb-1.4.2.tar.gz && tar -zxvf libmaxminddb-1.4.2.tar.gz && cd libmaxminddb-1.4.2 && ./configure && make && make check && make install && sudo ldconfig && cd .. && git clone https://github.com/maxmind/MaxMind-DB-Reader-php.git && cd MaxMind-DB-Reader-php/ext && /usr/bin/phpize && ./configure && make && make install && echo "extension=maxminddb.so" >> /etc/php/7.2/cli/php.ini && echo "extension=maxminddb.so" >> /etc/php/7.2/fpm/php.ini && sudo apt install openssl && curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && sudo apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget curl php7.2-curl && sudo npm install --global --unsafe-perm puppeteer && sudo chmod -R o+rx /usr/lib/node_modules/puppeteer/.local-chromium && sed 's|mozilla\/AddTrust_External_Root.crt|#mozilla\/AddTrust_External_Root.crt|g' -i /etc/ca-certificates.conf && sudo update-ca-certificates -f -v && wget -q -O - https://packages.blackfire.io/gpg.key | apt-key add - && echo "deb http://packages.blackfire.io/debian any main" | sudo tee /etc/apt/sources.list.d/blackfire.list && apt update -y && apt install blackfire-agent -y && version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$version && mkdir -p /tmp/blackfire && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire && mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so && printf "extension=blackfire.so\n" > /etc/php/7.2/cli/conf.d/blackfire.ini && printf "extension=blackfire.so\n" > /etc/php/7.2/fpm/conf.d/blackfire.ini && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz && curl -sO https://gordalina.github.io/cachetool/downloads/cachetool.phar && chmod +x cachetool.phar && mv cachetool.phar /usr/local/bin/cachetool && echo "nameserver 8.8.8.8" >> /etc/resolv.conf && echo "nameserver 8.8.4.4" >> /etc/resolv.conf && apt-get install dnsmasq -y && groupadd -r dnsmasq && composer self-update && composer self-update --2 && apt-get install jpegoptim -y && apt-get install optipng -y && apt-get install pngquant -y && npm install -g svgo && apt-get install gifsicle -y && apt-get install webp -y && apt-get remove -qq -y php7.2-dev pkg-config gcc libmagickwand-dev build-essential && apt-get auto-remove -qq -y && apt-get clean && rm -rf /etc/ImageMagick-6/policy.xml && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*' returned a non-zero code: 1
2020-11-05T09:56:03.2995648Z ##[error]Process completed with exit code 1.
2020-11-05T09:56:03.3079575Z Post job cleanup.
2020-11-05T09:56:03.4221004Z [command]/usr/bin/git version
2020-11-05T09:56:03.4310796Z git version 2.29.2
2020-11-05T09:56:03.4325765Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-11-05T09:56:03.4373689Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-11-05T09:56:03.4668260Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-11-05T09:56:03.4698236Z http.https://github.com/.extraheader
2020-11-05T09:56:03.4715424Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-11-05T09:56:03.4752193Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-11-05T09:56:03.5089392Z Cleaning up orphan processes
```

--- a/php/Dockerfile-php
+++ b/php/Dockerfile-php
@@ -4,8 +4,8 @@ FROM quay.io/continuouspipe/ubuntu16.04:${FROM_TAG}
 # Install PHP packages, including the Tideways extension
 ARG PHP_VERSION
 ENV PHP_VERSION=${PHP_VERSION:-7.0}
-RUN echo 'deb http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian main' > /etc/apt/sources.list.d/tideways.list \
- && curl -sS https://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages/EEB5E8F4.gpg | apt-key add - \
+RUN echo 'deb https://packages.tideways.com/apt-packages debian main' > /etc/apt/sources.list.d/tideways.list \
+ && curl -L -sS https://packages.tideways.com/key.gpg | apt-key add - \
  && if [ "$PHP_VERSION" != "7.0" ]; then \
    echo 'deb http://ppa.launchpad.net/ondrej/php/ubuntu xenial main' > /etc/apt/sources.list.d/php-ppa.list; \
    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E5267A6C; \
@@ -52,7 +52,7 @@ RUN echo 'deb http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian m
  # Install composer for PHP dependencies \
  && wget https://getcomposer.org/installer -O /tmp/composer-setup.php -q \
  && [ "$(wget https://composer.github.io/installer.sig -O - -q)" = "$(sha384sum /tmp/composer-setup.php | awk '{ print $1 }')" ] \
- && php /tmp/composer-setup.php --install-dir='/usr/local/bin/' --filename='composer' --quiet \
+ && php /tmp/composer-setup.php --install-dir='/usr/local/bin/' --filename='composer' --version='1.10.17' --quiet \
  && rm /tmp/composer-setup.php
 
 USER build

--- a/tideways/Dockerfile
+++ b/tideways/Dockerfile
@@ -1,8 +1,8 @@
 ARG FROM_TAG=latest
 FROM quay.io/continuouspipe/ubuntu16.04:${FROM_TAG}
 
-RUN echo 'deb http://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages debian main' > /etc/apt/sources.list.d/tideways.list \
- && curl -sS https://s3-eu-west-1.amazonaws.com/qafoo-profiler/packages/EEB5E8F4.gpg | apt-key add - \
+RUN echo 'deb https://packages.tideways.com/apt-packages debian main' > /etc/apt/sources.list.d/tideways.list \
+ && curl -L -sS https://packages.tideways.com/key.gpg | apt-key add - \
  && apt-get update -qq \
  && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install \
     tideways-daemon \

